File: 백준/Bronze/5565. 영수증/영수증.cc
#include <iostream>
using namespace std;

int main() {
	int totalPrice;
	cin >> totalPrice;

	int sum = 0;

	for (int i = 0; i < 9; i++)
	{
		int price;
		cin >> price;

		sum += price;
	}

	cout << totalPrice - sum << endl;
}

File: 백준/Bronze/5565. 영수증/영수증.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int totalPrice = sc.nextInt();

        int sum = 0;

        for (int i = 0; i < 9; i++) {
            int price = sc.nextInt();
            sum += price;
        }

        System.out.println(totalPrice - sum);
    }
}